samcli/commands/check/resources/LambdaFunctionPricing.py

```diff
@@ -0,0 +1,17 @@
+"""
+Object to contain Lambda function pricing info.
+This data gets stored in the graph, not a lambda function object
+"""
+
+
+class LambdaFunctionPricing:
+    number_of_requests: int
+    average_duration: int
+    allocated_memory: float
+    allocated_memory_unit: str
```

Comment:
same comment about class variables, and public

Reply:
I explained why in above comment